code/modules/clothing/spacesuits/plasmamen.dm

```diff
@@ -96,13 +100,28 @@
 				update_icon()
 		else
 			to_chat(user, "<span class='warning'>Seems like someone already drew something on this helmet's visor!</span>")
+		return
+	if(istype(hitting_item, /obj/item/clothing/head))
+		var/obj/item/clothing/hitting_clothing = hitting_item
+		if(hitting_clothing.clothing_flags & PLASMAMAN_HELMET_EXEMPT)
+			to_chat(user, "<span class='notice'>You cannot place [hitting_clothing.name] on the helmet!</span>")
```

Comment:
oh wait... should this one have "the"? idk why, but it felt ok to my brain without it, but I just noticed it, and I had to actually think about it a bit... what do you think?

Reply:
Yes, it should, almost forgot.